packages/rum/src/domain/assemblyV2.ts

```diff
@@ -63,7 +54,9 @@ export function startRumAssemblyV2(
           ? combine(rumContext, viewContext, actionContext, rawRumEvent)
           : combine(rumContext, viewContext, rawRumEvent)
         const serverRumEvent = withSnakeCaseKeys(rumEvent)
-        serverRumEvent.context = combine(savedGlobalContext || getGlobalContext(), customerContext)
+        const globalAttributes = savedGlobalAttributes || getGlobalAttributes()
+        serverRumEvent.context = combine(globalAttributes.context, customerContext)
+        serverRumEvent.user = deepClone(globalAttributes.user as Context)
```

Comment:
what is the reasoning for cloning the user but not the context?
it would probably be better to do it when setting the user to avoid to do it for each event.

Reply:
The context is cloned via `combine`. I had to do this to support [this test](https://github.com/DataDog/browser-sdk/blob/benoit/user-api/packages/rum/src/domain/assemblyV2.spec.ts#L194-L208) which is a copy of [this one about global context](https://github.com/DataDog/browser-sdk/blob/benoit/user-api/packages/rum/src/domain/assemblyV2.spec.ts#L140-L154), but I'm not sure why we need to dereference the global context in the first place, since it is serialized synchronously right after that. Maybe I could remove the `deepClone` and test?

Reply:
sounds good to me